lib/dip/config.rb

```diff
@@ -112,6 +113,24 @@ def to_h
       end
     end
 
+    def validate
+      raise Dip::Error, "Config file path is not set" if file_path.nil?
+      raise Dip::Error, "Config file not found: #{file_path}" unless File.exist?(file_path)
+
+      schema_path = File.join(File.dirname(__FILE__), "../../schema.json")
```

Comment:
Where is the top level key `$schema` used?

Reply:
It was only ment to be used by the editor for validation, not by dip. I'll remove and use `yaml-language-server: $schema=<urlToTheSchema>` (see #177) instead.

Reply:
1. Removed `$schema`
2. Updated the README about `yaml-language-server`